SW_VegProd.c

```diff
@@ -841,3 +865,541 @@ void get_critical_rank(void){
 		 End of rank_SWPcrits
 	 ----------------------------------------------------------*/
 }
+
+/**
+ @brief Wrapper function for estimating natural vegetation. First, climate is calculated and averaged, then values are estimated
+
+ @param[in,out] vegProd Structure holding all values for vegetation cover of simulation
+ @param[in] startYear Starting year of the simulation
+ @param[in] endYear Ending year of the simulation
+ @param[in] veg_method User specified value determining method of vegetation estimation with the current option(s):
+ 1 - Estimate fixed vegetation composition (fractional cover) from long-term climate conditions
+ */
+
+void estimateVegetationFromClimate(SW_VEGPROD *vegProd, int startYear, int endYear,

```

Comment:
Please add a test that calls `estimateVegetationFromClimate()` with `veg_method = 1`